zebrad/src/commands/start/sync.rs

```diff
@@ -79,13 +82,16 @@ where
         // Query the current state to construct the sequence of hashes: handled by
         // the caller
         //
-        // TODO(jlusby): get the block_locator from the state
+        // TODO(jlusby): get the real network
```

Comment:
I can fix this up as part of #717, once #699 merges.

Reply:
```suggestion
        // TODO(teor): get the real network
```